### PR TITLE
document env vars for module analysis cache

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
# PR Summary

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [x] Version 7.x preview content
- [x] Version 7.0 content
- [x] Version 6 content
- [x] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

File: reference/5.1/Microsoft.PowerShell.Core/About/about_Environment_Variables.md [hunk 1]
```diff
@@ -235,6 +235,41 @@ The environment variables that store preferences include:
 
     For more information, see [about_Execution_Policies](about_Execution_Policies.md).
 
+- PSModuleAnalysisCachePath
+
+  PowerShell provides control over the file that is used to cache data about a
+  module, such as the commands it exports.
+
+  By default on Windows, this cache is stored in the file
+  `${env:LOCALAPPDATA}\Microsoft\Windows\PowerShell\ModuleAnalysisCache`. The cache is typically read
```
Comment (AnatoliB) [inline reference/5.1/Microsoft.PowerShell.Core/About/about_Environment_Variables.md]: Strictly speaking, this is not entirely correct. For example, on the Azure App Service sandbox, the `$env:LOCALAPPDATA` points to a certain existing location (`D:\local\LocalAppData`), but this is not where PowerShell creates the cache files. Instead, it takes the location from `Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData)` (returns an empty string in on the sandbox environment).

Comment (AnatoliB) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Environment_Variables.md]: So, in PowerShell terms, the location is: 
``` PowerShell
`$([System.Environment]::GetFolderPath([System.Environment+SpecialFolder]::LocalApplicationData))\Microsoft\Windows\PowerShell\ModuleAnalysisCache`
```
I realize this looks a bit verbose for documentation, but this is the truth.

By the way, if the code is changed to _actually_ use `$env:LOCALAPPDATA`, the https://github.com/Azure/azure-functions-powershell-worker/issues/365 issue will also be solved.

Comment (SteveL-MSFT) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Environment_Variables.md]: I discussed this with Sean and updated it.  The technically correct path is too long and confusing to users and isn't really applicable to most users as the env var is correct except in the sandbox.

Comment (AnatoliB) [reply reference/5.1/Microsoft.PowerShell.Core/About/about_Environment_Variables.md]: Agreed. The word "normally" is very appropriate here.

**Pull request merged**